MoveImage2/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Text;
+using Form;
+
+namespace Form
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.Write("Input console color: ");
+
+            string foregroundColor = Console.ReadLine();
+            Color consColor = new Color(foregroundColor);
+            consColor.GetConsoleColor();
+
+            Console.Write("Choose your image type(1-Rectangle, 2-Triangle): ");
+            string str = Console.ReadLine();
+            int.TryParse(str, out int type);
+            IBuilder builder;
+            if (type == 1)
```

Comment:
Not correct checking. How I can understand that if type == 1 then its  rectangle by reading a code?